webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsActions.js

```diff
@@ -10,7 +10,21 @@ import {
   SAVE_UPSTREAM_SUBSCRIPTIONS_REQUEST,
   SAVE_UPSTREAM_SUBSCRIPTIONS_SUCCESS,
   SAVE_UPSTREAM_SUBSCRIPTIONS_FAILURE,
-} from './UpstreamSubscriptionsContstants';
+  PING_UPSTREAM_SUBSCRIPTIONS_SUCCESS,
+  PING_UPSTREAM_SUBSCRIPTIONS_FAILURE,
+} from './UpstreamSubscriptionsConstants';
+
+export const pingUpstreamSubscriptions = () => async (dispatch) => {
```

Comment:
I debated whether or not to have this function do anything with redux, but it seemed to fit in nicely and it is consistent with all other actions